### PR TITLE
Added SLIs dashboard

### PR DESCRIPTION
**Pull request by jhellar**
## What

Added SLIs dashboard.

## Why

For SREs to have overview of different SLIs.

## How

For each SLI there is a row with the indicator and a graph with historical data.

screenshot1
screenshot2

### CODE REVIEW

File: install/resources/monitoring-cluster/dashboards/Makefile [hunk 1]
```diff
@@ -6,5 +6,6 @@ all:
 	@cat ./strimzi-kafka-mirror-maker-2.yaml | sed -e 's/<namespace>/$(PER_CLUSTER_GRAFANA_NAMESPACE)/g' | cat | oc apply -f -
 	@cat ./strimzi-operators.yaml | sed -e 's/<namespace>/$(PER_CLUSTER_GRAFANA_NAMESPACE)/g' | cat | oc apply -f -
 	@cat ./strimzi-zookeeper.yaml | sed -e 's/<namespace>/$(PER_CLUSTER_GRAFANA_NAMESPACE)/g' | cat | oc apply -f -
+	@cat ./slis.yaml | sed -e 's/<namespace>/$(PER_CLUSTER_GRAFANA_NAMESPACE)/g' | cat | oc apply -f -
```
Comment (Rajagopalan-Ranganathan) [inline install/resources/monitoring-cluster/dashboards/Makefile:9]: Can we rename  the file to be aligned with the existing file names like strimzi-sli may be? WDYT?

**Comment by Rajagopalan-Ranganathan** [comment]
We have increased the zookeeper and kafka replicas to 3, i am not sure if that is causing the issue. But in altest master with your changes I am getting the below error in the Dashboard.
image

File: install/resources/monitoring-cluster/dashboards/slis.yaml [hunk 2]
```diff
@@ -0,0 +1,1839 @@
...
+            "full": false,
+            "lineColor": "rgb(31, 120, 193)",
+            "show": false,
+            "ymax": null,
+            "ymin": null
+          },
+          "tableColumn": "",
+          "targets": [
+            {
+              "expr": "kube_pod_status_ready{namespace=\"kafka-cluster\", condition=\"true\", pod=~\".*kafka.*\"}",
```
Comment (Rajagopalan-Ranganathan) [inline install/resources/monitoring-cluster/dashboards/slis.yaml:135]: the namespace names looks like are hard coded, that might be a problem. we should try to follow the pattern in other dashboards with templating: ``` {
            "allValue": null,
            "current": {},
              "definition": "",
            "hide": 0,
            "includeAll": false,
            "label": "Namespace",
            "multi": false,
            "name": "kubernetes_namespace",
            "options": [],
            "query": "query_result(kafka_server_replicamanager_leadercount)",
            "refresh": 1,
            "regex": "/.*namespace=\"([^\"]*).*/",
            "skipUrlSync": false,
            "sort": 0,
            "tagValuesQuery": "",
            "tags": [],
            "tagsQuery": "",
            "type": "query",
            "useTags": false
          },``` refer : strimzi-kafka.yaml to get the required namespace names.

Comment (jhellar) [reply install/resources/monitoring-cluster/dashboards/slis.yaml]: Should be fixed now.

**Comment by jhellar** [comment]
Noticed that the `latency` metrics are reseted to 0ms when kafka broker pods restart, so the metric is not correct in that case. ping @david-martin just fyi

File: install/resources/monitoring-cluster/dashboards/Makefile (hunk 1 above)
Comment (jhellar) [reply install/resources/monitoring-cluster/dashboards/Makefile:9]: Not sure. The `strimzi-.*` dashboards come from strimzi operator examples, right? This one does not, that is why I left it like this.

Comment (Rajagopalan-Ranganathan) [reply install/resources/monitoring-cluster/dashboards/Makefile:9]: slis for what? could be added to the name. WDYT?

**Comment by Rajagopalan-Ranganathan** [comment]
I have not tested the changes yet, the changes for hard coded namespaces looks good. need to test before merging.

**Comment by Rajagopalan-Ranganathan** [comment]
> cc @jhellar @david-martin @pb82 We have increased the zookeeper and kafka replicas to 3, i am not sure if that is causing the issue. But in latest master with your changes I am getting the below error in the Dashboard.
> image

@jhellar did you a chance to look at this?

**Comment by jhellar** [comment]
@Rajagopalan-Ranganathan yes, the issue with 3 replicas should be fixed now.

File: install/resources/monitoring-cluster/dashboards/Makefile (hunk 1 above)
Comment (jhellar) [reply install/resources/monitoring-cluster/dashboards/Makefile]: ok, makes sense, added `strimzi-kafka-` prefix.

**Comment by pb82** [comment]
@jhellar will create a follow up issue for the reset on broker restarts.

**Pull request merged**